charts/github-bot-safe-settings/Chart.yaml

```
@@ -12,5 +12,5 @@ home: https://github.com/github/safe-settings
 sources:
   - https://github.com/github/safe-settings
 maintainers:
-  - name: Yadhav Jayaraman
-    email: decyjphr@github.com
+  - name: Saritasa
+    email: nospam@saritasa.com
```

This repository is private, I don't think we'll get any spam here.
```suggestion
    email: devops@saritasa.com
```

---

done